<?xml version="1.0"?>
<launch>
  <arg name="robot_config" default="thor"/>

  <node name="motion_service" pkg="motion_editor_core" type="motion_service" respawn="false" output="screen">
    <param name="robot_config" value="$(arg robot_config)"/>
  </node>
</launch>
